import-export-cli/api_meta.yaml

```diff
@@ -0,0 +1,7 @@
+deploy:
```

Comment:
This file is also not from the code. Isn't it?

Reply:
yes, it was removed